sysutils/libvirt/Portfile

```diff
@@ -152,7 +152,12 @@ variant qemu description {Local QEMU support} {
     startupitem.name        ${name}
     startupitem.pidfile     ${prefix}/var/run/libvirtd.pid
     startupitem.start       ${prefix}/sbin/libvirtd
+}
 
+variant hvf description {Experimental macOS hypervisor support} requires qemu {
+    # this patch is a rebase the hvf-domain branch from
+    # https://github.com/roolebo/libvirt -- and it appears to work!
+    patchfiles-append       hvf.diff
```

Comment:
The commit message should be "libvirt: add hvf variant" (not "libvirt: add qemu variant").

I am not very interested in being responsible for rebasing this patch forever. If this patch is useful, it should be upstream. What's the status of that?

Until this patch is upstream, I could leave updates of libvirt and py-libvirt to you so that you can update this patch when needed. Or I could continue to update libvirt and py-libvirt but ignore this patch and if it breaks you can update it then.

Reply:
The patch isn't really all that good, and given that `virt-manager` would require a fair amount of hacks to make this vaguely usable, I think I'll park it for now, and reduce all of this to something more manageable.